### PR TITLE
added vita balance details

### LINKED ISSUE #983: Add VITA Balance in Explorer
image

### PR DESCRIPTION
**Pull request by gyanpgupta**
**What type of PR is this?**
/kind ui-change
**What this PR does / why we need it**:
To add vita balance in address page
**Which issue(s) this PR fixes**:
Fixes #983

### CODE REVIEW

**Comment by gyanpgupta** [comment]
VitaBalance

File: src/shared/address-details/address-details.tsx [hunk 1]
```diff
@@ -72,6 +75,10 @@ class AddressDetailsInner extends PureComponent<Props, State> {
     }
     const xrc20tokens =
       isBrowser && JsonGlobal("state").base.defaultERC20Tokens;
+    const vitaToken = isBrowser && JsonGlobal("state").base.vitaTokens[0];
```
Comment (puncsky) [inline src/shared/address-details/address-details.tsx]: please use props

**Comment by gyanpgupta** [comment]
@puncsky  @nmhung1210  please have a look now

File: src/shared/address-details/address-details.tsx [hunk 2]
```diff
@@ -99,6 +108,13 @@ class AddressDetailsInner extends PureComponent<Props, State> {
         )).toFixed(4)} IOTX`}</div>
         {this.renderOtherTokenBalance()}
       </div>
+      <div className={"item"}>
```
Comment (nmhung1210) [inline src/shared/address-details/address-details.tsx]: Let's try to use GRID for this.

File: src/client/stylesheets/main.scss [hunk 3]
```diff
@@ -75,7 +75,7 @@
     overflow: hidden;
     margin-bottom: 20px;
     .item {
-      width: 33.3%;
+      width: 25%;
```
Comment (nmhung1210) [inline src/client/stylesheets/main.scss]: It's not good in term of responsive (small screen devices for example).

**Comment by gyanpgupta** [comment]
AddressTop

**Comment by gyanpgupta** [comment]
Address2Top

**Comment by gyanpgupta** [comment]
Address3Top

**Comment by nmhung1210** [comment]
Closed due to new UI merged at https://github.com/iotexproject/iotex-explorer/pull/903

**Pull request closed without merging**